src/components/Layout/Layout.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { NavLink, Outlet } from 'react-router-dom';
+import classNames from 'classnames';
+
+export const Layout: React.FC = () => (
+  <div>
```

Comment:
```suggestion
  <>
```
Тут краще б використати фрагмент, бо семантичні header і main, загорнуті в div - не дуже семантичний і валіднй код

Reply:
Логічно)) Інколи пальці самі друкують, а писати код, звісно, потрібно з розумом і уважно:face_with_monocle: